accelerator/conn_mqtt.c

```diff
@@ -32,7 +32,7 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
-  if (ta_config_set(&ta_core.cache, &ta_core.service) != SC_OK) {
+  if (ta_config_set(&ta_core) != SC_OK) {
```

Comment:
At conn_mqtt.c, we call `ta_config_set` without calling `ta_config_destory` when cleanup while we do call `ta_config_destory` at main.c and server.cc.
Is there a reason? @HowJMay

Reply:
I think it's my bad